ci/citest/acceptance-test/tests/cmd_console_test.go

```diff
@@ -21,6 +22,23 @@ func runConsoleCmd(instance_id string, t *testing.T) {
 	RunCmdAndExpectFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -- false", instance_id))
 }
 
+func runConsoleCmdWithPrivatekey(instance_id string, private_key_path string, t *testing.T) {
+	RunCmdAndReportFail(t, "openvdc", "console", instance_id, "-i", private_key_path)
+	RunCmdAndReportFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s ls", instance_id, private_key_path))
+	RunCmdAndReportFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s -- ls", instance_id, private_key_path))
+	RunCmdAndExpectFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s -- false", instance_id, private_key_path))
+}
```

Comment:
You should also confirm that the console command fails when you provide the wrong private key.

Reply:
Added TestLXCCmdConsole_AuthenticationPubkeymethod testing worng key.